res/css/views/groups/_GroupPublicityToggle.scss

```diff
@@ -20,23 +20,16 @@ limitations under the License.
     margin: 8px;
 }
 
-.mx_GroupPublicity_toggle > label {
-    display: flex;
-    align-items: flex-start;
-}
-
-.mx_GroupPublicity_toggle > label,
 .mx_GroupPublicity_toggle .mx_GroupTile {
-    width: 50%;
-}
-
-.mx_GroupPublicity_toggle input {
-    margin-right: 8px;
-    vertical-align: -4px;
+    width: calc(100% - 60px); // 48px switch + some padding
```

Comment:
Does adding `box-sizing: border-box;` and then using `width: 100%` allow you to avoid this calc? (I have already sprinkled `box-sizing: border-box;` on many of my Fields so far...)

Reply:
Not sure if it will work as well for toggle switches as it seems to for the field inputs I've used....

Reply:
That seems to do it - I'll apply it here. Thanks!